src/popup/router/components/AssetSelector.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <select
+    class="asset-selector"
+    :value="JSON.stringify(currentToken)"
+    @change="(event) => handleChange(event.target.value) "
+  >
+    <option
+      v-for="token in tokenList"
+      :key="token.contractId"
+      :value="JSON.stringify(token)"
+    >
+      {{ token.symbol }}
+    </option>
+  </select>
+</template>
+
+<script>
+import { mapState, mapGetters } from 'vuex';
+
+export default {
+  computed: {
+    ...mapState('fungibleTokens', ['availableTokens', 'aePublicData', 'selectedToken']),
+    ...mapGetters('fungibleTokens', ['tokenBalances']),
+    currentToken() {
+      return this.selectedToken || this.aeternityToken;
+    },
+    /**
+     * Returns the default aeternity meta information
+     */
+    aeternityToken() {
+      return this.aePublicData && Object.keys(this.aePublicData).length > 0
+        ? {
+          ...this.aePublicData,
+          convertedBalance: this.tokenBalance,
+          symbol: 'AE',
+          balanceCurrency: this.balanceCurrency,
+          contractId: 'aeternity',
+        }
+        : null;
+    },
+    /**
+     * Converts the token information object into a searchable list
+     */
+    convertedTokenInfo() {
+      const tokens = Object.entries(this.availableTokens).map(([contractId, tokenData]) => ({
+        name: tokenData.name,
+        symbol: tokenData.symbol,
+        contractId,
+        decimals: tokenData.decimals,
+        convertedBalance: tokenData.convertedBalance,
+      }));
+      this.tokenBalances.forEach((b) => {
```

Comment:
what is b what i t ? pls set name of this variable

Reply:
Same here - not my code.